ray-operator/test/support/environment.go

```diff
@@ -20,7 +22,11 @@ func GetRayVersion() string {
 }
 
 func GetRayImage() string {
-	return lookupEnvOrDefault(KuberayTestRayImage, RayImage)
+	rayImage := lookupEnvOrDefault(KuberayTestRayImage, RayImage)
+	if runtime.GOARCH == "arm64" && !strings.HasSuffix(rayImage, "-aarch64") {
+		rayImage = rayImage + "-aarch64"
```

Comment:
Add a log when explicitly modifying the user's image

Reply:
I have removed the arm chip support part after reconsideration. Thank you for the review!